Source/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can this be part of `Build\Blazorise.props`? Or is `Directory.Build.props` some special rule for VS to pick up when building projects?

Reply:
VS walks up the directory tree and looks for Directory.Build.props